mqt/benchviewer/src/backend.py

```diff
@@ -150,14 +150,66 @@ def createDatabase(zip_file: ZipFile):
     return database
 
 
-def read_mqtbench_all_zip():
+def read_mqtbench_all_zip(
+    skip_question: bool = False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+):
     global MQTBENCH_ALL_ZIP
-    huge_zip = Path("./static/files/qasm_output/MQTBench_all.zip")
+    huge_zip = Path(target_location + "/MQTBench_all.zip")
+    version_of_suitable_benchmark_zipfile = "v1.0.0"
+
+    url = (
+        "https://api.github.com/repos/nquetschlich/test/releases/tags/"
+        + version_of_suitable_benchmark_zipfile
+    )
+    response = requests.get(url)
+    if not response:
+        print("Suitable benchmarks cannot be downloaded, URL is faulty.")
+    else:
+        if not skip_question:
+            file_size = int((response.json()["assets"][0]["size"])) / 10e6
+            response = input(
+                "Shall the benchmarks file suitable with this MQTBench ({} MB) version be downloaded? (Y/n)".format(
```

Comment:
I believe this still needs a bit of work to be accessible for users. 
First of all, when starting the server, users should be informed of what is happening.
Something like 
- "Searching for suitable benchmark collection..."
- "Found local benchmark collection." In the case that a zip file is already present
- "No local collection found. Querying GitHub..."
- "Found `MQTBench.zip` (Version x.y.z, Size XXXX MB) [weblink]
- "Would you like to download the corresponding file? (Y/n)"
- (progress messages)
- "Download complete."
- (starting server message)